src/WireMock.Net/Settings/WireMockServerSettings.cs

```diff
@@ -234,6 +234,19 @@ public class WireMockServerSettings
         [PublicAPI]
         public bool CustomCertificateDefined => CertificateSettings?.IsDefined == true;
 
+#if USE_ASPNETCORE
+        /// <summary>
+        /// Client certificate mode for the server
+        /// </summary>
+        [PublicAPI]
+        public ClientCertificateMode ClientCertificateMode { get; set; }
+
+        /// <summary>
+        /// Whether to skip client certificate validation
+        /// </summary>
+        public bool SkipClientCertificateValidation { get; set; }
```

Comment:
I do understand this property.

Just an idea: would a better name be `AcceptAnyServerCertificate` ?
Like I used here:
https://github.com/StefH/RestEase.Authentication.Azure/blob/main/src/RestEase.Authentication.Azure/Options/AzureAuthenticatedRestEaseOptions.cs#L32


(Just an idea.....)

Reply:
I renamed to use the same naming convention, but it's really the server's validation of the client certificate, to I replaces Server with Client, `AcceptAnyClientCertificate`.

Reply:
OK. Sounds good.